projects/packages/stats/tests/php/test-xmlrpc-provider.php

```diff
@@ -92,4 +113,14 @@ public function test_get_blog() {
 
 		$this->assertSame( $expected_stats_blog, $stats_blog );
 	}
+
```

Comment:
This looks to be duplicated from parent class

Reply:
Nice catch, removed!